deploy/helm/values/staging-mtr.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for cfe-civil staging
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 5
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 500Mi
+  requests:
+    cpu: 100m
+    memory: 125Mi
+
+ingress:
+  className: modsec
+  hosts:
+    - cfe-civil-staging-mtr.cloud-platform.service.justice.gov.uk
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "cfe-civil-cfe-civil-staging-mtr-green"
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/enable-modsecurity: "true"
+    nginx.ingress.kubernetes.io/modsecurity-snippet: |
+      SecRuleEngine On
+
+deploy:
+  host: cfe-civil-staging-mtr.cloud-platform.service.justice.gov.uk
+
+postgresql:
+  enabled: false
+
+threshold:
+  useTestData: 'true'
+  futureFile: 'mtr-2026.yml'
+
+legalFrameworkApi:
+  host: http://legal-framework-api.legal-framework-api-production.svc.cluster.local
```

Comment:
For this to work: https://github.com/ministryofjustice/cloud-platform-environments/pull/21063